### PR TITLE
Components/common

### PR DESCRIPTION
**Pull request by fstodulski**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -16,13 +18,16 @@ interface IAppProps {
 class App extends React.Component<IAppProps, {}> {
   componentDidMount() {
     getAsset(2, 1).then((result: IAsset) => {
-      this.props.stores.assets.avatar.filePath = `http://localhost:8081/${result.filePath}`;
+      this.props.stores.assets.avatar.filePath = `http://localhost:8081/${
```
Comment (stanimirtt) [inline src/App.tsx]: Use env variable.

File: src/assets/styles/common.scss [hunk 2]
```diff
@@ -0,0 +1,15 @@
+.container {
+  max-width: 1224px;
+  margin: 0 auto;
+}
+
```
Comment (stanimirtt) [inline src/assets/styles/common.scss:5]: remove extra line.

File: src/components/Common/Button/index.ts [hunk 3]
```diff
@@ -1,3 +1,2 @@
-import Button from './Button';
-
-export { Button };
+export { DefaultButton } from './Default-button/Default-button';
```
Comment (stanimirtt) [inline src/components/Common/Button/index.ts]: Let's stick to Pascal case - './DefaultButton/DefaultButton';

File: src/components/UserSettingsModal/form-models/describtion.ts [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import { IDefaultForm } from '../../common/forms/DefaultForm/DefaultForm.interface';
```
Comment (stanimirtt) [inline src/components/UserSettingsModal/form-models/describtion.ts]: What you think to move interfaces to `/interfaces` folder?

File: src/components/UserSettingsModal/form-models/index.ts [hunk 5]
```diff
@@ -0,0 +1,3 @@
+export { securityPasswordForm } from './security-password';
```
Comment (stanimirtt) [inline src/components/UserSettingsModal/form-models/index.ts:1]: No dashesh.

File: src/components/UserSettingsModal/form-models/index.ts [hunk 6]
```diff
@@ -0,0 +1,3 @@
...
+export { userDetailsForm } from './user-details';
+export { describtionForm } from './describtion';
```
Comment (stanimirtt) [inline src/components/UserSettingsModal/form-models/index.ts]: typo b -> p

File: src/App.tsx [hunk 7]
```diff
@@ -1,11 +1,13 @@
 import * as React from 'react';
 import './App.scss';
+import './assets/styles/common.scss';
 
 import MainLayout from './components/Layouts/MainLayout';
 
 import { getAsset } from './services/asset';
 import { IAsset } from './interfaces/asset';
 import { observer, inject } from 'mobx-react';
+import { UserSettingModal } from './components/UserSettingsModal';
```
Comment (suezzo) [inline src/App.tsx]: Typo, the file name should be the same as the exported component name.

File: src/components/UserSettingsModal/form-models/user-details.ts [hunk 8]
```diff
@@ -0,0 +1,31 @@
+import { IDefaultForm } from './../../common/forms/DefaultForm/DefaultForm.interface';
```
Comment (suezzo) [inline src/components/UserSettingsModal/form-models/user-details.ts]: Would be better to move this interface to `src/interfaces`

File: src/components/common/forms/DefaultForm/DefaultForm.interface.ts [hunk 9]
```diff
@@ -0,0 +1,5 @@
+import { IDefaultInput } from './../../inputs/default/Default-input.interface';
+export interface IDefaultForm {
+  formTitle: string;
+  inputFields: IDefaultInput[];
+}
```
Comment (suezzo) [inline src/components/common/forms/DefaultForm/DefaultForm.interface.ts]: Would be better to move this interface to `src/interfaces`

**Review by stanimirtt — APPROVED** [review]

**Review by suezzo — APPROVED** [review]

**Pull request merged**